packages/nextra-theme-docs/src/constants.tsx

```diff
@@ -126,7 +126,7 @@ export const themeSchema = z
       labels: z.string()
     }),
     sidebar: z.object({
-      defaultMenuCollapseLevel: z.number().min(2).int(),
+      defaultMenuCollapseLevel: z.number().min(0).int(),
```

Comment:
```suggestion
      defaultMenuCollapseLevel: z.number().min(1).int(),
```
this was my mistake 🙈, but it should be 1 (because behaviour is the same for 1 and 0 - collapse all)

Reply:
set to `1`